db/upgrade-X.X.X-X.Y.Z.sql

```diff
@@ -316,3 +316,15 @@ DROP INDEX mac ON violation;
 --
 
 DROP INDEX ip on inline_accounting;
+
+--
+-- Alter table iplog_history
+--
+
+ALTER TABLE iplog_history ADD INDEX end_time (end_time) ADD INDEX start_time (start_time);
```

Comment:
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'ADD INDEX start_time (start_time)' at line 1

Missing comma before second ADD